kie-dmn-feel/src/main/java/org/kie/dmn/feel/util/Either.java

```diff
@@ -46,6 +46,6 @@ public boolean isRight() {
     
     public <X> X cata(Function<L,X> left, Function<R,X> right) {
         // warning: left.invoke, because of FEEL specs, could return null. The below is the safest way to implement cata over this Either.
-        return isLeft() ? left.apply( this.left.get() ) : right.apply( this.right.get() );
+        return isLeft() ? left.apply( this.left.get() ) : right.apply( this.right.orElse( null ) );
```

Comment:
@tarilabs I am not sure if this is the best way to solve this problem, but the fact is that functions and decision tables can correctly return null as a result and we need to support this case.